This patch fixes the error that occurs during the linkage
phase of xsltproc binary. Symbols xsltCopyTree and xsltDefaultTrace
cannot be externally referenced from libxslt.so library if they
are defined as static. This patch simply removes the static keywords
to fix the compilation issues.

The patch was sent to upstream:
https://mail.gnome.org/archives/xslt/2019-February/msg00003.html

--- libxslt-1.1.33/libxslt/transform.c	2019-01-28 07:30:24.483271722 +0000
+++ libxslt-1.1.33/libxslt/transform.c	2019-01-28 07:19:57.302639856 +0000
@@ -90,7 +90,7 @@ int xsltMaxVars = 15000;
 static xmlNsPtr
 xsltCopyNamespaceListInternal(xmlNodePtr node, xmlNsPtr cur);
 
-static xmlNodePtr
+xmlNodePtr
 xsltCopyTree(xsltTransformContextPtr ctxt, xmlNodePtr invocNode,
 	     xmlNodePtr node, xmlNodePtr insert, int isLRE,
 	     int topElemVisited);
@@ -469,7 +469,7 @@ xsltGetXIncludeDefault(void) {
     return(xsltDoXIncludeDefault);
 }
 
-static unsigned long xsltDefaultTrace = (unsigned long) XSLT_TRACE_ALL;
+unsigned long xsltDefaultTrace = (unsigned long) XSLT_TRACE_ALL;
 
 /**
  * xsltDebugSetDefaultTrace:
@@ -1618,7 +1618,7 @@ occupied:
  *
  * Returns a pointer to the new tree, or NULL in case of error
  */
-static xmlNodePtr
+xmlNodePtr
 xsltCopyTree(xsltTransformContextPtr ctxt, xmlNodePtr invocNode,
 	     xmlNodePtr node, xmlNodePtr insert, int isLRE,
 	     int topElemVisited)
